client/src/components/VerbContainer.jsx

```diff
@@ -62,89 +160,104 @@ export default class VerbContainer extends Component {
     }
   }
 
-  toggleTense(name) {
-    this.setState((prevState) => {
-      const newState = {
-        tenses: {
-          ...prevState.tenses,
-          [name]: !prevState.tenses[name]
-        }
-      }
-      localStorage.setItem(
-        'tenses',
-        JSON.stringify(Object.assign(prevState.tenses, newState.tenses))
-      );
-      return newState;
-    });
+  toggleTense(tenseNames) {
+    if (tenseNames instanceof Array) {
```

Comment:
you can use default props to set tenseNames as an empty array to start so you don't have to check this...